pkg/version/version_test.go

```diff
@@ -143,14 +147,24 @@ func TestDefaultVersionResolver_StableVersion(t *testing.T) {
 				FileUtilService:  tt.fields.FileUtilService,
 				HttpClient:       tt.fields.HttpClient,
 			}
-			got, err := r.StableVersion()
+			got, err := r.StableVersion(true)
```

Comment:
```suggestion
			got, err := r.StableVersion(tt.fields.fromCache)
```
and remove if !tt.fields.fromCache {

Reply:
Great idea! Thanks